### PR TITLE
Development invert read keys

### PR DESCRIPTION
**Pull request by logotie**
In reference to #5 

With this one, I wasn't particularly to sure how to do it.

I understood as:
When the config is active.
Keynext should now be remapped to key previous
Keyprevious should now be remapped to key next.

image

Please let me know, if there's any changes you want made!

### CODE REVIEW

File: src/main/resources/bundles/Bundle_en.properties [hunk 1]
```diff
@@ -47,6 +47,7 @@ hue_prompt=Hue:
 id_prompt=ID:
 install_update_plugins=Install/Update Plugins
 installed_plugins=Installed Plugins
+invert_reading_style=Invert the reading keys for page next/previous
```
Comment (xgi) [inline src/main/resources/bundles/Bundle_en.properties]: Change this to "Read pages right-to-left"

File: src/main/java/com/faltro/houdoku/model/Config.java [hunk 2]
```diff
@@ -70,6 +70,12 @@ public Object getValue(Field field) {
          * potentially not retrieving all of a series' chapters.
          */
         QUICK_RELOAD_SERIES("quick_reload_series", true),
+        /**
+         * Inverts the reading keys, this is to simulate a more traditional reading style
+         * where on pressing the left key, it goes to the next page
+         * and on pressing the right key, it goes to the previous page
+         */
+        INVERT_READING_STYLE("invert_reading_style", false),
```
Comment (xgi) [inline src/main/java/com/faltro/houdoku/model/Config.java]: You can keep the name of this config option.

Comment (logotie) [reply src/main/java/com/faltro/houdoku/model/Config.java]: Thanks for taking the time out to review my changes! :)  For this one should I do it like:
INVERT_READING_STYLE("INVERT_READING_STYLE", false)?

Apologies I think I'm misunderstanding it

Comment (xgi) [reply src/main/java/com/faltro/houdoku/model/Config.java]: Sorry, no, you can keep it like you have it. I was just letting you know you don't need to change the name of the setting even though I asked you to change the wording in the UI.

**Comment by logotie** [comment]
> I'd also like to have the buttons be inverted when the config setting is enabled. You can modify the first/next/previous/lastPage methods in the ReaderController to utilize the config setting (and probably change the names of the methods to be more generic). You'd also need to look at refreshPage to disable appropriate buttons.

Can I please ask why would I need to change the first and last page methods please?  I think I'm misunderstanding haha

I also need to look at refreshPage?

**Comment by xgi** [comment]
The goal is to make it so that when the "invert" setting is enabled, the effect of the buttons should swap as well. That is, pressing the left button would go to the next page and the right button would go to the previous page.

Currently, the buttons are set up so that the left button calls the prevPage method and the right button calls the nextPage method. What I'd like is for i.e. the left button to instead call a leftPage method which goes to the previous page by default, but goes to the next page if the invert setting is enabled.

**Comment by logotie** [comment]
Ah thank you so much, I understand now.

I never thought about the actual buttons on the app.

**Comment by logotie** [comment]
I have been having some issues.  When the keys are inverted, it does not automatically proceed to the next chapter.  This is what I am trying to fix.

**Comment by logotie** [comment]
I have got it working, I think it might need some refactoring

**Pull request merged**